lib/archiving.js

```diff
@@ -49,6 +49,13 @@ var request = require('request'),
 *          <a href="https://dashboard.tokbox.com">OpenTok dashboard</a>.
 *   </ul>
 *
+* @param {Boolean} hasAudio
```

Comment:
do you want to say that this parameter is optional and that it will default to true?
just not to confuse people thinking these are new mandatory fields.